src/popup/lib/store.ts

```diff
@@ -0,0 +1,18 @@
+import { writable } from 'svelte/store'
+import browser from 'webextension-polyfill'
+import type { Locale } from '../../types'
+
+export const locale = writable<Locale>(null)
+
+browser.storage.local.get('locale').then((result) => {
```

Comment:
What happens if the popup is open but the tab changes? How does the popup get notified of locale updates?

Reply:
I though about that, but since the popup is the only one that can change that it should not be an issue.
Nothing updates the locale outside of the popup